groovy/src/main/java/org/apache/zeppelin/groovy/GroovyInterpreter.java

```diff
@@ -55,159 +55,172 @@
 
 /**
  * Groovy interpreter for Zeppelin.
+ * @author dlukyanov@ukr.net / dmitry lukyanov
```

Comment:
code autoformat.. but it's not allowed to keep the `created by` ? any footprint? )))